docs/encrypt-logs.md

```diff
@@ -0,0 +1,61 @@
+# Log encryption
+
+Describes the redo log encryption and the undo log encryption.
+
+## Redo Log encryption
+
+Use the `innodb_redo_log_encrypt` option to enable or disable redo log data encryption. By default, the encryption of the redo log is disabled.
```

Comment:
Likewise, how are our `innodb_redo_log_encrypt ` and `innodb_undo_log_encrypt` different from the upstream?

Reply:
One of our documentation initiatives is that we have complete docs.